src/main/features/ccj/views/date-of-birth.njk

```diff
@@ -0,0 +1,41 @@
+{% extends "layout.njk" %}
+{% from "includes/macros/form.njk" import csrfProtection, textInput, radioGroup, saveAndContinueButton, errorSummary %}
+{% from "includes/macros/date.njk" import dateInput %}
+
+{% set title = t('Money claim') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <form method="post">
+        {{ csrfProtection(csrf) }}
+
+        <h1 class="heading-large">
+          <span class="heading-secondary">{{ t('County Court Judgment') }}</span>
+          {{ t("Defendant's date of birth") }}
+        </h1>
+
+        <p>{{ t("We use this to ensure legal proceedings are made against the right person. " +
+        "For example, if there are 2 people living at the same address with the same " +
+        "name.") }}</p>
+
+        {{ radioGroup(
```

Comment:
Actually your right, it isn't forcing the user to select one, so I'll look into it